### PR TITLE
fix: re-assign original method back to be able to re-use in the future

### PR DESCRIPTION
**Pull request by arturovt**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngneat/until-destroy/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #66 

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: integration/app/app.component.spec.ts [hunk 1]
```diff
@@ -220,4 +220,61 @@ describe('until-destroy runtime behavior', () => {
...
+    let originalStopCalledTimes = 0;
+
+    @Injectable()
+    class IssueSixtySixService {
+      start(): void {
+        startCalledTimes++;
+
+        new Subject()
+          .pipe(
+            untilDestroyed(this, 'stop'),
```
Comment (NetanelBasal) [inline integration/app/app.component.spec.ts]: I just notice that we're using a `string` for the method type. We can make it safer by using the `keyof` type.

Comment (arturovt) [reply integration/app/app.component.spec.ts]: Good idea. Should it be in a separate PR?

Comment (NetanelBasal) [reply integration/app/app.component.spec.ts]: No, let's add it here. Thanks!

Comment (arturovt) [reply integration/app/app.component.spec.ts]: @NetanelBasal ping

Comment (NetanelBasal) [reply integration/app/app.component.spec.ts]: 💯

**Comment by NetanelBasal** [comment]
Please release a new version.

**Comment by NetanelBasal** [comment]
What is your npm user name?

**Comment by arturovt** [comment]
> What is your npm user name?

`overthesanity`

**Comment by NetanelBasal** [comment]
You should now have permission to publish. LMK if it works.

**Comment by arturovt** [comment]
> You should now have permission to publish. LMK if it works.

It worked. And I forgot to copy readme into folder before publishing... :rofl: 

Have to publish another patch... :)

**Pull request merged**